app/components/Form/ApplicationForm.tsx

```diff
@@ -442,13 +453,19 @@ const ApplicationForm: React.FC<Props> = ({
       } else if (!isSubmitEnabled && !projectAreaAccepted) {
```

Comment:
Additionally, I know this was probably my mistake but just a logic error I saw here, I think this `else if` here should be `!isSubmitted` and not `!isSubmitEnabled` since that will also trigger for orgName not being filled out, etc.

We can fix it here/now or we can just open it in another ticket.

Reply:
yes, we can handle that change here.